### ISSUE TITLE
physics watcher/freeze not working

### ISSUE THREAD

**Issue by leasoncre** [SBPrime/AsyncWorldEdit-Premium#31]
i've had this issue from the same time i've had the chunk relight issue. but i've been neglecting to post a separate ticket about it.

git-Spigot-5f38d38-18fbb24 (MC: 1.8.8) (Implementing API version 1.8.8-R0.1-SNAPSHOT)
Async v3.2.3 (was running 3.2.2 prior to updating about an hour ago, and re-testing the issue)
WorldEdit version 6.1;no_git_id

in short: physics watcher/freeze is not functional

have tried changing the config setting and reloading config / restarting server. still doesn't work. sand/gravel(etc) still fall immediately after pasting

**Comment by SBPrime** [comment]
I tested the physics freeze a moment ago and it looks ok.

Please do this test:
- start a new test server that has only WE + AWE (flat creative)
- make this structure on the ground:
  
  ```
  SSSS
  SWWS
  SWWS
  SSSS
  
  S - Stone
  W - Watter
  ```
- Copy the structure (only 1 layer!)
- Paste it in air

If the water spills out the PhysicsFreeze is not working if it stays its ok.

I did exactly the same test. With PF on the water stayed in place, without it spilled all arround the map.

**Comment by leasoncre** [comment]
try it with sand, gravel, anvil or flowing water/lava (ids: 12,13,145,8,10). as regular minecraft will not update still water/lava (ids: 9,11) until an adjacent block is updated

tested as described above on new server. STILL water/lava is fine.

**Comment by SBPrime** [comment]
I'm not sure if the scenario that you are describing ever worked. I always tested it using watter and it was intended for watter and lava. I'll try to change it so it works like you described. To be 100% sure what do you need, are you talking about keeping the sand from falling during the edit or after the edit has finished?

**Comment by leasoncre** [comment]
in older versions, the physics blocks would stay suspended until a player hand placed a block to update the physics blocks. ie: as if they had been generated by MC (ie: gravel in the nether) and would remain suspended until a player came along and updated it.

so probably hold the blocks suspended during the operation, and until the operation is complete. (in theroy) they should remain suspended until manually updated (and not updated by another world edit operation) 

in short: exactly like previous versions of AWE (i think 3.1.1 and earlier, been a while, so not sure on exactly what older version it worked)

**Comment by SBPrime** [comment]
Done.
